ExRam.Gremlinq.Providers.CosmosDb/GremlinQuerySourceExtensions.cs

```diff
@@ -134,5 +134,26 @@ public static IConfigurableGremlinQuerySource UseCosmosDb(this IConfigurableGrem
                         source.Logger)
                     .UseGraphsonDeserialization(new TimespanConverter()));
         }
+
+        public static IConfigurableGremlinQuerySource UseCosmosDbEmulator(this IConfigurableGremlinQuerySource source, string database, string graphName, string authKey, string hostname = "localhost", int port = 8901)
```

Comment:
That would just work fine, thanks a lot. It is also a great chance for some code reuse ;-)

Reply:
I updated this method for code reuse, is this better?